onnxruntime/core/providers/nv_tensorrt_rtx/nv_provider_factory.cc

```diff
@@ -91,6 +88,8 @@ struct NvProviderFactory : IExecutionProviderFactory {
   ~NvProviderFactory() override {}
 
   std::unique_ptr<IExecutionProvider> CreateProvider() override;
+  std::unique_ptr<IExecutionProvider> CreateProvider(const OrtSessionOptions& session_options,
```

Comment:
I think this function is never used ?

Reply:
Ok, it seems this is to address the review comment about future use with Compile API. makes sense.